extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -96,6 +96,36 @@
     @ConfigItem(name = "auth.session.encryption-key")
     public Optional<String> encryptionKey;
 
+    /**
+     * Enable socket reuse port
+     */
+    @ConfigItem(defaultValue = "false", name = "so-reuse-port")
+    public boolean soReusePort;
```

Comment:
Why have you changed the default to false? I thought the usual default value here would be true?

Reply:
It's a new configuration. In this case one that enables [SO_REUSEPORT](https://lwn.net/Articles/542629/). At a first glance I figured it safer to leave this new configurations turned off. 

All the new configurations here only take effect if native transport is enabled. I'll add that to the documentation.

Reply:
Sorry, I should have read it more closely, I was thinking it was SO_REUSEADDR

Reply:
No worries. I added this to the docs.